WMF Framework/Wikipedia.xcdatamodeld/Wikipedia 2.xcdatamodel/contents

```diff
@@ -70,6 +70,7 @@
         </fetchIndex>
     </entity>
     <entity name="TalkPage" representedClassName="TalkPage" syncable="YES" codeGenerationType="class">
+        <attribute name="dateAccessed" attributeType="Date" defaultDateTimeInterval="582233760" usesScalarValueType="NO" syncable="YES"/>
```

Comment:
This defaults to the date/time I created the attribute (06/14/2019), let me know if there's a better one.